public/js/components/Frames.js

```diff
@@ -66,6 +66,9 @@ const Frames = React.createClass({
 
   renderFrames() {
     let { frames } = this.props;
+    if (!frames) {
+      return null;
+    }
```

Comment:
If you want to do some small cleanup here we can open a new small PR. 

I'd also be curious to see what adding flow types would look like in a component. We haven't done that yet